go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/pions/webrtc
 
 require (
+	github.com/gorilla/websocket v1.4.0
```

Comment:
can you put this dependency in the examples directory, I don't want to make this mandatory to users who don't want this library.

Reply:
Good idea, i will fix.